src/Microsoft.ML.LightGBM/LightGbmArguments.cs

```diff
@@ -92,14 +95,13 @@ private static string GetArgName(string name)
         [BestFriend]
         internal static class Defaults
         {
-            [BestFriend]
-            internal const int NumBoostRound = 100;
+            public const int NumBoostRound = 100;
         }
 
         public sealed class TreeBooster : BoosterParameter<TreeBooster.Arguments>
         {
             public const string Name = "gbdt";
-            public const string FriendlyName = "Tree Booster";
+            internal const string FriendlyName = "Tree Booster";
 
             [TlcModule.Component(Name = Name, FriendlyName = FriendlyName, Desc = "Traditional Gradient Boosting Decision Tree.")]
             public class Arguments : ISupportBoosterParameterFactory
```

Comment:
>Arguments [](start = 25, length = 9)

Can you rename this to `Options`? you would also need to rename the local variable of type Arguments, they are usually called `args` and we usually rename to `options`.

Reply:
There is already a class named Options at line 43 above.

---
In reply to: [256654714](https://github.com/dotnet/machinelearning/pull/2476#discussion_r256654714) [](ancestors = 256654714)

Reply:
Could you name it with something that's related to `Options`? Before the general renaming of `Arguments` to `Options` both this and the above at line 43 were named `Arguments`, and it wasn't an issue.  But I understand this is a more specific `Options` class, that's why it could make sense to add a prefix to the name. 

---
In reply to: [256671757](https://github.com/dotnet/machinelearning/pull/2476#discussion_r256671757) [](ancestors = 256671757,256654714)